R/subscript.R

```diff
@@ -9,57 +9,65 @@
 #' not checked in any way (length, missingness, negative elements).
 #'
 #' @inheritParams vec_as_location
+#' @param indicator,location,name How to handle indicator (logical),
+#'   location (numeric), and name (character) subscripts. If
+#'   `"coerce"`, the subscript is coerced to its base type,
```

Comment:
I found the coercion wording a bit hard to understand — I think you need a sentence describing the common case (i.e. it's a base vector) before talking about S3 vectors.